lib/curator/services/factory_service.rb

```diff
@@ -85,7 +85,7 @@ def find_or_create_nomenclature(nomenclature_class:, term_data: {}, authority_co
             Rails.logger.error '==============================================='
             Rails.logger.error e.backtrace.join("\n")
             Rails.logger.error '==============================================='
-            raise e
+            raise
```

Comment:
intentional to call `raise` with no args?

Reply:
This actually re raises the previous error according to this answer 
https://stackoverflow.com/a/23771227